assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -62,12 +63,26 @@ const MIN_MAX = {
     MIN: 1,
     MAX: 1000,
   },
+  X: {
+    MIN: 0,
+    MAX: PAGE_WIDTH,
+  },
+  Y: {
+    MIN: 0,
```

Comment:
Note that `0` for `y` is actually the safezone limit. So the very top of the Page is `0 - SAFE_ZONE_AREA`